common.gypi

```diff
@@ -172,6 +172,16 @@
         },
         'cflags': [ '-O3' ],
         'conditions': [
+          ['node_section_ordering_file!="-"', {
```

Comment:
Since `common.gypi` is also used to compile addons any variables referenced needs to have default values so addons can be compiled with older versions of Node.js where the variable isn’t present. See https://github.com/nodejs/node/pull/33688 for an example.